src/Components/Profile/ProfilePhoto.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
No need to import React a top each file, this is no longer a requirement since later versions of React.